kuma/dashboards/views.py

```diff
@@ -71,8 +73,19 @@ def revisions(request):
             start_date = end_date - datetime.timedelta(seconds=seconds)
             query_kwargs['created__range'] = [start_date, end_date]
 
-    if query_kwargs:
-        revisions = revisions.filter(**query_kwargs)
+        authors_filter = filter_form.cleaned_data['authors']
+        if authors_filter not in ['', 'All Authors']:
```

Comment:
If the requester asks for a username, I think they should get it, and skip the known/unknown authors filter